lightkurve/periodogram.py

```diff
@@ -371,6 +508,40 @@ def flatten(self, method='logmedian', filter_width=0.01, return_trend=False):
             return snr, bkg
         return snr
 
+    def _get_fwhm(self, numax):
+        """For a given numax, the method calculates the expected Full Width Half
+        Maximum of the seismic mode envelope as (Mosser et al 2010) for RGB
+        stars:
+
+        fwhm = 0.66 * numax^0.88,
+
+        if the data is generated for a long-cadence timeseries. If the
+        timeseries is short-cadence, the relation will be:
+
+        fwhm = 0.25 * numax.
+
+        This will be based on the highest frequency in the periodogram.
```

Comment:
This sentence confuses me a bit.  The equations above have fwhm depend on numax, but this sentence suggests it is based on max(self.frequency).  Can you clarify and/or edit this sentence?